utils/blech_post_process_utils.py

```diff
@@ -34,6 +34,8 @@ def __init__(self, sort_file_path):
             sort_table.sort_values(
                     ['len_cluster','Split'],
                     ascending=False, inplace=True)
+            if 'level_0' in sort_table.columns:
```

Comment:
`level_0` is added by the .reset_index() function: I guess when pandas resets the index, it creates that column as a record of the new index.

The issue is, that it then permanently writes that column to the output csv file. If you're using the csv as the input for cell sorting, you create it manually, yes, but then on the first run through post_process, it writes in that column. If you run the same CSV through post-process AGAIN, it tries to write `level_0` in again, and can't write it overtop of the existing `level_0`, which throws an error.

So this is a little bit of a niche problem; you need to be running post_process with a csv input, and then re-run the same csv to create the error. I mainly ran into the issue as a result of testing my code as I familiarize myself with the pipeline. That said, I could imagine scenarios where you accidentally added the wrong cell to the spreadsheet, or you're not happy with a split/merge outcome, or something similar, and want to run post_process again, and if you're using the csv input (which I like a lot, being very pro-automation), then this saves you from needing to manually go in and delete `level_0` from the spreadsheet.